components/filesystem/lock_handler.rst

```diff
@@ -62,8 +62,23 @@ You can pass an optional blocking argument as the first argument to the
 ``lock()`` method, which defaults to ``false``. If this is set to ``true``, your
 PHP code will wait indefinitely until the lock is released by another process.
 
-The resource is automatically released by PHP at the end of the script. In
-addition, you can invoke the
-:method:`Symfony\\Component\\Filesystem\\LockHandler::release` method to release
-the lock explicitly. Once it's released, any other process can lock the
-resource.
+Beware that the resource lock is automatically released as soon as PHP applies the
+garbage-collection process to the ``LockHandler`` object. This means that if you
+refactor the first example showed in this article as follows:
+
+.. code-block:: php
+
+    use Symfony\Component\Filesystem\LockHandler;
+
+     if (!(new LockHandler('hello.lock'))->lock()) {
```

Comment:
also, I don't like this new syntax. It's too much stuff in one line and works only in >=PHP 5.4